packages/components/src/navigation/item/use-navigation-tree-item.tsx

```diff
@@ -22,7 +27,9 @@ export const useNavigationTreeItem = ( itemId, props ) => {
 	useEffect( () => {
 		const isMenuActive = activeMenu === menu;
 		const isItemVisible =
-			! search || normalizedSearch( props.title, search );
+			! search ||
+			( props.title !== undefined &&
+				normalizedSearch( props.title, search ) );
```

Comment:
⚠️ Runtime change:


Looking at `normalizedSearch`, I assumed that it doesn't accept `undefined` arguments:
- the function to which it's inspired in `packages/block-editor/src/components/inserter/search-items.js` also doesn't accept `undefined`
- the `removeAccents` external utility function also doesn't seem to accept empty strings

Therefore, given the previous assumption, I've added a check on `props.title` — on purpose, I made the check explicit for `undefined`, since `normalizedSearch( '', '' )` should be returning `true`.